app/routes/encrypt-all-user-lnmsecrets.tsx

```diff
@@ -0,0 +1,15 @@
+import { useLoaderData } from "@remix-run/react";
+import { json } from "@remix-run/node";
+import { prisma } from "~/db.server";
+import { encryptAllSecrets } from "~/models/user.server";
+
+export async function loader() {
```

Comment:
when does this get called?

Reply:
When we go to that route - /encrypt-all-user-lnmsecrets

I will delete this route after we run it.